endor/src/error/TokenNotFoundException.js

```diff
@@ -0,0 +1,7 @@
+import NotFoundException from './NotFoundException';
+
+export default class TokenNotFoundException extends NotFoundException {
+  constructor(message) {
+    super(message, 'Not Found', 404);
```

Comment:
I could be wrong here. But a TokenNotFoundException would mean that the token is not valid, meaning the user on the request would not be authenticated. Should this be a 401?